src/components/Formik/Forms/AdminViewPasswordChangeForm.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {Formik, Form, FormikHelpers} from 'formik';
+import {Input} from "../Input/Input";
+import * as Yup from "yup";
+import {Button} from "../../Button/Button";
+
+interface Props {
+    handleModalExit: () => void;
+}
+
+interface Password {
+    password: string;
+    passwordCheck: string;
+}
+
+export const AdminViewPasswordChangeForm = (props: Props) => {
+
+
+    return (
+
+        <Formik initialValues={
+            {
+                password: '',
+                passwordCheck: '',
+            }}
+                validationSchema={Yup.object({
+                    password: Yup.string()
+                        .required("Pole wymagane")
+                        .min(8, "Hasło musi mieć minimum 8 znaki")
+                        .matches(/[A-Z]+/, 'Jeden duży znak')
+                        .matches(/\d+/, 'Numer')
+                        .matches(/[@$!%*#?&]+/, "Jeden znak specjalny"),
```

Comment:
Messeges which you write in Yup are displayed to client on front - Right now they don't inform client properly about details of problem

For example - rename "Jeden duży znak" na "Hasło musi zawierać jeden duży znak" 

On the other hand - i think that best will be to combine it into one RegEx and display one message to client " Hasło musi zawierać minimum jedną dużą literę, jedną cyfrę i jeden znak specjalny"